src/main/java/edu/kis/powp/jobs2d/transformations/changedriver/ApplyTransformationToNewDriver.java

```diff
@@ -0,0 +1,32 @@
+package edu.kis.powp.jobs2d.transformations.changedriver;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.ImmutableCompoundCommand;
+import edu.kis.powp.jobs2d.command.manager.ICommandManager;
+import edu.kis.powp.jobs2d.events.SelectClearPanelOptionListener;
+import edu.kis.powp.jobs2d.features.*;
+
+public class ApplyTransformationToNewDriver {
+
+
+    public static void setUpTransformationToNewDriver() {
+        RecordCommandsForTransformation.setupRecorder();
+    }
+    public static void clearRecordCommands() {
+        RecordCommandsForTransformation.clear();
+    }
+
+    public static void applyTransformation(Job2dDriver transformation) {
+
+        DrawerFeature.getDrawerController().clearPanel();
+        LinesRecorder.getLinesRecorder().clearLines();
+
+        ICommandManager manager = CommandsFeature.getCommandManager();
+        manager.setCurrentCommand(RecordCommandsForTransformation.getRecordedCommand());
+
+        DriverCommand command = CommandsFeature.getCommandManager().getCurrentCommand();
+        command.execute(DriverFeature.getDriverManager().getCurrentDriver());
```

Comment:
My main concern regarding this solution is that it uses commands manager for refreshing/restarting the transformation. It seems totally unnecessary, and makes harder to use commands manager. 
Please remove any CommandManager usages in your solution.